pkg/utils/gnmiutils.go

```diff
@@ -299,10 +299,14 @@ func UpdateForElement(value interface{}, path string, pathParams ...string) (*gn
 		update.Val.Value = &gnmi.TypedValue_IntVal{IntVal: reflect.Indirect(reflectValue).Int()}
 	case "*bool":
 		update.Val.Value = &gnmi.TypedValue_BoolVal{BoolVal: reflect.Indirect(reflectValue).Bool()}
+	case "*float32", "*float64":
```

Comment:
@SeanCondon  The only thing that almost worked is to handle both float32 and float64 with TypedValue_FloatVal. The sra hits this case.  But it rounds up the number. We need to handle both float32 and float64 via DoubleVal  in onos-api, onos-config, and aether-roc-api. But I wasn't able to make them work tonight.